charts/kubeit-app-of-apps/templates/applications.yaml

```diff
@@ -99,15 +109,13 @@ spec:
       ignoreMissingValueFiles: {{ ($app.chart | default dict ).ignoreMissingValueFiles | default $.Values.globals.chart.ignoreMissingValueFiles }}
       skipCrds: {{ ($app.chart | default dict ).skipCrds | default $.Values.globals.chart.skipCrds }}
       valueFiles:
-        - ./values.yaml
-        - ./appValues/global.yaml
-        - ./appValues/{{ $app.name }}/appGlobal.yaml
-        - ./appValues/{{ $app.name }}/environment/{{ $envDirectory }}/{{ $envDirectory }}.yaml
-        - ./appValues/{{ $app.name }}/environment/{{ $envDirectory }}/region/{{ $.Values.region }}/{{ $.Values.region }}.yaml
-        - ./appValues/{{ $app.name }}/environment/{{ $envDirectory }}/region/{{ $.Values.region }}/cluster/{{ $.Values.clusterColour }}.yaml
-        - ./appValues//{{ $app.name }}/environment/{{ $envDirectory }}/region/{{ $.Values.region }}/cluster/namespaces/{{ . }}.yaml
+        - "{{ $.Values.globals.valuesPath }}/global.yaml"
+        {{- range $valuesList }}
+        - "{{ . }}"
+        {{- end }}
       values: |
```

Comment:
Add a variable which would contain a list with nonempty valuesFiles to see which ones were actually loaded